citation/caching.py

```diff
@@ -6,10 +6,10 @@
 from django.db import connection
 from django.db.models import Count
 
-from catalog.core.views import generate_network_graph_group_by_tags, generate_network_graph_group_by_sponsors, \
+from citation.views import generate_network_graph_group_by_tags, generate_network_graph_group_by_sponsors, \
```

Comment:
generally prefer relative imports when possible, so this would be `from .views`. We should also move `generate_network_graph_group_by_tags` into a standalone module however, they don't really belong in the view as far as I can tell..